it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/TemplateTestBase.java

```diff
@@ -625,7 +625,7 @@ protected String getGcsPath(String artifactId, GcsResourceManager gcsResourceMan
 
   /** Create the default configuration {@link PipelineOperator.Config} for a specific job info. */
   protected PipelineOperator.Config createConfig(LaunchInfo info) {
-    return createConfig(info, null);
+    return createConfig(info, Duration.ofMinutes(30));
```

Comment:
Should we make this an else condition here instead? That way all instances of this method pick up the change

https://github.com/GoogleCloudPlatform/DataflowTemplates/blob/fb29227731a8a351163a37f4da1c2a51dd156125/it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/TemplateTestBase.java#L636-L638

Reply:
Also, can we make this 45 minutes? The example you screenshotted ran for almost 30 minutes

Reply:
ACK.